packages/wonder-blocks-grid/components/grid.js

```diff
@@ -30,7 +30,7 @@ type Props = {
     spec: GridSpec,
 
     /** The `<Row>` components that will make up the grid */
-    children: React.ChildrenArray<Row>,
+    children: React.Node,
```

Comment:
Why is this so general now?

Reply:
I realized that it was far too strict - if anyone wanted to make a component that returned a Row it wouldn't work - so I just deferred to the docs to encourage it instead.

Reply:
Yeah, that's something that's pretty hard to model.  It's not like the component is inheriting from `Row` and yet it's still technically a `Row`.  I wonder if we could change this to be:
```
children: React.ChildrenArray<
   Row | React.Component<{render(): Row}> | () => Row
>
```

Reply:
We do need it to be a `Row` or a component that renders a `Row` for this to work ,right?

Reply:
Nope! The Grid component doesn't actually render anything, it just sets the context. I did it before to try and provide some encouragement but honestly a grid can hold anything - and it can even be set way up the tree and only later have the row be used.